<launch>
    <node ns="led" pkg="ros_ws281x" name="ros_ws281x_node" type="ros_ws281x_node" clear_params="true" output="screen">
        <param name="target_frequency" value="800000"/>
        <param name="gpio_pin" value="21"/>
        <param name="dma" value="10"/>
        <param name="strip_type" value="WS2811_STRIP_GBR"/>
        <param name="led_count" value="3"/>
        <param name="invert" value="false"/>
        <param name="brightness" value="255"/>
    </node>
</launch>
